app/org/sagebionetworks/bridge/stormpath/StormpathAccount.java

```diff
@@ -201,6 +229,7 @@ private void encryptJSONTo(String key, Object value) {
     private void encryptTo(String key, String value) {
         if (value == null) {
             acct.getCustomData().remove(key);
+            acct.getCustomData().remove(key+VERSION_SUFFIX);
```

Comment:
We can check to see if the attribute is removed from the mock customData. Added to some tests in StormpathAccountTest.